content/en/tracing/advanced_usage/_index.md

```diff
@@ -2029,17 +2029,7 @@ var tracer = Datadog.Trace.Tracer.Create(isDebugEnabled: true);
 Debug mode is disabled by default. To enable it, set the environment variable `DD_TRACE_DEBUG=true`. See the PHP [configuration docs][1] for details about how and when this environment variable value should be set in order
 to be properly handled by the tracer.
 
-**Application Logs**:
-
-By default, logging from the PHP tracer is disabled. In order to get debugging information and errors sent to logs,
-set a [PSR-3 logger][2] singleton.
-
-```php
-\DDTrace\Log\Logger::set(
-    new \DDTrace\Log\PsrLogger($logger)
-);
-```
-
+The location of the log file depends on your setup: it can be either ErrorLog directive (apache), error_log directive (nginx) or error_log ini param in php/php-fpm.
```

Comment:
Not completely sure what this sentence means. By "directive", do you mean directory? Also, having trouble understanding the last part: what do you mean by the error_log ini param in php/php-fpm?

Reply:
Hi! In order to tell PHP where it should put error_log messages you can either set it at the server level or as a PHP `ini` parameter, which is the standard way to configure php behavior.

If you are using Apache as a server, then you can use the Apache's `ErrorLog` directive.
If you are using nginx as a server, then you can use the nginx's `error_log` directive.
If you are configuring instead at php level, you can use the php's `error_log` ini parameter.

Reply:
Got it! Let's go with:

```suggestion
To configure the location of your error log messages, set it at the server level (use the `ErrorLog` directive in Apache or the `error_log` directive in NGINX) or at the PHP level, using PHP's `error_log` ini parameter.
```